.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
 python:
   - "2.7"
+
+before_install:
+  - wget https://web.cimne.upc.edu/users/croig/data/kratos-simphony.tgz -O /tmp/kratos.tgz
+  - mkdir $PWD/kratos
+  - tar -xzvf /tmp/kratos.tgz -C $PWD/kratos
+  - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/kratos/libs
+  - export PYTHONPATH=$PYTHONPATH:$PWD/kratos
+  - export PATH=$PATH:$PWD/kratos
+
 install:
   - pip install -r dev_requirements.txt
   - python setup.py develop
 
 script:
-  - flake8 --exclude=CFD_example.gid,3balls.gid --ignore=F401,F403 .
\ No newline at end of file
+  - flake8 --exclude=use_cases,utilities,kratos,ProjectParameters.py,DEM_explicit_solver_var.py --ignore=F403 .
```

Comment:
you should add the specifics into a configuration file, see http://flake8.readthedocs.org/en/latest/config.html#global